containers/ecr-viewer/src/app/tests/services/ecrSummaryService.test.tsx

```diff
@@ -200,6 +200,64 @@ describe("Evaluate eCR Summary Condition Summary", () => {
     ).not.toBeInTheDocument();
     expect(screen.getByText("SARS-CoV-2 PCR")).toBeInTheDocument();
   });
+  it("should return immunization details based on snomed code", () => {
+    const actual = evaluateEcrSummaryConditionSummary(
+      BundleEcrSummary as unknown as Bundle,
+      mappings,
+    );
+    render(
+      actual[1].immunizationDetails.map((detail) => (
+        <React.Fragment key={Math.random()}>{detail.value}</React.Fragment>
+      )),
+    );
+
+    expect(screen.getByText("SARS-CoV-2 PCR Vaccine")).toBeInTheDocument();
+  });
+  it("should not display non-related immunization details", () => {
+    const actual = evaluateEcrSummaryConditionSummary(
+      {
+        ...BundleEcrSummary,
+        entry: [
+          ...BundleEcrSummary.entry,
+          {
+            fullUrl: "urn:uuid:6689c3f5-f256-9c28-bd98-89905630f28d",
+            resource: {
+              resourceType: "Immunization",
+              vaccineCode: {
+                coding: [
+                  {
+                    code: "24",
+                    system: "urn:oid:2.16.840.1.113883.12.292",
+                    display: "anthrax",
+                  },
+                ],
+              },
+              extension: [
+                {
+                  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/condition-code",
+                  coding: [
+                    {
+                      code: "722545003",
+                      system: "http://snomed.info/sct",
+                    },
+                  ],
+                },
+              ],
+            },
+          },
+        ],
+      } as unknown as Bundle,
+      mappings,
+    );
+    render(
+      actual[1].immunizationDetails.map((detail) => (
+        <React.Fragment key={Math.random()}>{detail.value}</React.Fragment>
+      )),
+    );
+
+    expect(screen.getByText("SARS-CoV-2 PCR Vaccine")).toBeInTheDocument();
+    expect(screen.queryByText("rabies")).not.toBeInTheDocument();
```

Comment:
should this say anthrax? I agree that rabies shouldn't be there, but I don't see any immunizations for rabies in the bundle

Reply:
Good catch, was going to use rabies at one point but changed it to anthrax since we have a good example of that. 
```suggestion
    expect(screen.queryByText("anthrax")).not.toBeInTheDocument();
```